src/services/TransactionService.ts

```diff
@@ -312,7 +312,7 @@ export class TransactionService extends AbstractService {
    * @param {Account} account
    * @return {SignedTransaction[]}
    */
-  public signStagedTransactions(account: Account): SignedTransaction[] {
+  public signStagedTransactions(account: any): SignedTransaction[] {
```

Comment:
why `any` ? Does it need to be a `PublicAccount` ? Then please use the `|` operator here, we don't want `any` types.